requirements/test.txt

```diff
@@ -18,6 +18,8 @@ django-filter==0.15.3
 django-model-utils==3.0.0
 django-object-actions==0.10.0
 django-simple-history==1.9.0
+django-waffle==0.12.0
+django==1.10.7            # via django-config-models, django-model-utils, edx-drf-extensions
```

Comment:
It looks like this is failing the tests. Did this get auto-generated or did you manually add it?